pkgs/servers/sql/postgresql/ext/pgrouting.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, postgresql, perl, cmake, gcc, boost, gmp, cgal, mpfr }:
+
+stdenv.mkDerivation rec {
+  pname = "pgrouting";
+  version = "2.6.2";
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ postgresql perl gcc boost gmp cgal mpfr ];
```

Comment:
gcc is provided by the stdenv so you don't need to mention it in buildInputs.
Also perl in not referenced in the build products so I imagine it is a build-time dependency which belongs to nativeBuildInputs (please check, nevertheless).

Reply:
All true. Corrected.